app/templates/me/index.hbs

```diff
@@ -31,8 +31,8 @@
 
     <p>
         If you want to use package commands from the command line, you will need to
-        login with <code>cargo login</code> using one of the tokens listed below. 
-        For security reasons, you should supply a token when prompted rather than passing it on the command line. 
+        login with <code>cargo login</code> using one of the tokens listed below. <br/>
+        When working in shared environments, it is more secure to supply a token when prompted rather than passing it on the command line. 
```

Comment:
Someone could see the running process or the remnants of it for a few seconds...and if your token was on the command line they'd see the argument and your token.

Working on the package-lock file